docs/hugo/content/introduction/diagnosing-problems-with-aso.md

```diff
@@ -0,0 +1,49 @@
+# Diagnosing problems with ASO
+
+## Common mistakes
+
+### ASO controller pod in Status CreateContainerConfigError
+```
+$ kubectl get pods -n azureserviceoperator-system
+NAME                                                       READY   STATUS                       RESTARTS   AGE
+azureserviceoperator-controller-manager-69cbccd645-4s5wz   1/2     CreateContainerConfigError   0          7m49s
+```
+
+Very likely that you forgot to create the `aso-controller-settings` secret. This secret must be in
+the same namespace as the pod.
+
+## Problems with resources
+
+### Resource with no Ready condition set
+When resources are first created a ready condition should appear quickly, indicating that the resource is being reconciled:
+```
+$ kubectl get resourcegroups.resources.azure.com 
+NAME            READY     SEVERITY   REASON          MESSAGE
+aso-sample-rg   False     Info       Reconciling     The resource is in the process of being reconciled by the operator   
+```
+if this isn't happening then check the [controller logs](#getting-aso-controller-pod-logs).
+
+
+## Getting ASO controller pod logs
+The last stop when investigating most issues is to look at the ASO pod logs. We expect that
+most resource issues can be resolved using the resources .status.conditions without resorting to 
+pod logs. Setup issues on the other hand may requiring digging into the ASO controller pod logs.
+
+Assuming that ASO is installed into the default namespace, you can look at the logs for the controller
+with the following command: 
+`kubectl logs -n azureserviceoperator-system azureserviceoperator-controller-manager-69cbccd645-9pdsv manager`
```

Comment:
Could use the label to get pods here so they don't need to work out the pod name themselves?
```suggestion
`kubectl logs -n azureserviceoperator-system --selector control-plane=controller-manager --container manager`
```

Reply:
Done